portals/publisher/source/src/app/components/Apis/Details/ProductResources/ProductResourcesEditWorkspace.jsx

```diff
@@ -438,7 +438,10 @@ function ProductResourcesEdit(props) {
                 const {
                     body: { list },
                 } = response;
-                const filteredList = list.filter((theApi) => theApi.lifeCycleStatus !== 'PROTOTYPED');
+                const filteredList = list.filter((theApi) => theApi.lifeCycleStatus !== 'PROTOTYPED' &&
+                    (theApi.lifeCycleStatus !== 'RETIRED') &&
```

Comment:
I think we can remove this extra brackets surrounding !== statements

Reply:
https://github.com/wso2/apim-apps/pull/134/commits/284d896f9d7ca99ec0727131f25ba991c0e82e89